.github/workflows/lint.yml

```
@@ -0,0 +1,37 @@
+name: Lint
+
+on:
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  htmlproofer:
+    name: Run htmlproofer
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Ruby
+        uses: actions/setup-ruby@v1
+        with:
+          ruby-version: 2.7.x
+      - uses: actions/cache@v2
+        with:
+          path: vendor/bundle
+          key: ${{ runner.os }}-gems-${{ hashFiles('**/Gemfile.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-gems-
+      - name: Bundle install
+        run: |
+          bundle config path vendor/bundle
+          bundle install
+      - run: bundle exec jekyll build
+      - name: Cache HTMLProofer
+        id: cache-htmlproofer
+        uses: actions/cache@v2
+        with:
+          path: tmp/.htmlproofer
+          key: ${{ runner.os }}-htmlproofer
+      - run: echo "::add-matcher::.github/workflow-support/htmlproofer-matcher.json"
+      - name: Check for broken links
+        run: bundle exec htmlproofer --assume-extension ./_site
+      - run: echo "::remove-matcher owner=htmlproofer::"
```

```suggestion
      - run: echo "::remove-matcher owner=htmlproofer::"
        if: success() || failure()
```
It's perhaps not required, but I think it's best to always make sure you clean up after yourself